heron/tracker/src/python/version.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Twitter. All rights reserved.
```

Comment:
can you move this common area - if everybody is interested in adding version information? heron-tracker, ui, cli, explorer, packer, etc.